src/content/docs/new-relic-one/use-new-relic-one/workloads/use-workloads.mdx

```diff
@@ -22,13 +22,22 @@ It comprises the following:
 
 The **Activity** tab shows performance data related to the entities in the workload, along with the events that could explain any changes in those time series. It looks like this:
 
-![workload_activity_tab.png](./images/workloads_activity_tab.png "workloads_activity_tab.png")
+![workload_activity_tab.png](./images/workloads20210526_activity.png "workloads_activity_tab.png")
 
 Here's the most important sections:
 1. Linked dashboards. You can add links to dashboards from your workload, and [create pre-filtered, workload-relevant links to dashboards](/docs/new-relic-one/use-new-relic-one/workloads/use-workloads/#add-dashboards).
 2. Golden metrics. These are charts with the most relevant metrics for each entity type, such as number of requests, response time, and error rate for an application. Explore the charts to detect correlations among different entities (for example, two applications) and different stack layers (for example, applications and hosts). The golden metrics that you see for each entity type on a workload can be customized either at the account or the workload level through the [NerdGraph API](/docs/apis/nerdgraph/examples/golden-metrics-entities-nerdgraph-api-tutorial/#customize-golden-metrics).
 3. Events timeline. This includes the start and end time of incidents and anomalies that refer to the workload entities. It also shows other event types that can explain a change in the status or performance of the workload, such as deployments and configuration changes.    
 
+The **Owner** tab gives you information about the team responsible for the workload. It looks like this:
+
+![workload_owner_tab.png](./images/workloads20210526_owner.png "workloads_owner_tab.png")
+
+It contains:
```

Comment:
Thanks @urbiz-nr ! I'm adding some suggestions to guide users a little bit more, a be more explicit about what they can do in the tab. 

It contains:
1. The team that is responsible for the workload. You can include more than one team.
2. The workload description. Share the mission of the workload, and the business logic it represents: Is it a web application? An API? A backend process? Include any details that are relevant to your team, or to other teams in your organization.
3. Contact information. From the drop-down menu, choose how you team prefers to be contacted.
4. Links to the most relevant resources to operate the workload. Here you can add links to runbooks, code repositories, productivity tools, or any other piece of documentation you want at hand concerning the workload.

Reply:
Thanks @helenapm - I'm adding your feedback.